mapDownload.properties

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
Haha. Good point. It does make me thing that the 'triplea_maps.xml' belongs outside of this main code repo though. It is more of a map configuration file and its sort of confusing for the github link to reference the code repo rather than some general map or config repo (sort of like how in SF the file is currently hosted in the map area not main code base). Thoughts?